features/topology-map/plugins/org.opennms.features.topology.plugins.topo.linkd/src/main/java/org/opennms/features/topology/plugins/topo/linkd/internal/LinkdStatusProviderAbstract.java

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2013-2014 The OpenNMS Group, Inc.
```

Comment:
This is a new file, introduced in 2019. This should be reflected here.

Reply:
yup

Reply:
18e27a7df4b..3ceab6d9ac7  jira/NMS-12410 -> jira/NMS-12410